### PR TITLE
Create email_notifier.py

### PR DESCRIPTION
**Pull request by tokevinm**

### CODE REVIEW

File: Folder/email_notifier.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+import smtplib
+import requests
+from dotenv import load_dotenv
+
+
+class EmailNotifier:
```
Comment (lawjonathan17) [inline Folder/email_notifier.py:7]: You could add a comment explaining what `EmailNotifier` is used for. Comments help people read the code and better understand. Also even helps future you when you try to remember wtf you did 6 months ago

File: Folder/email_notifier.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    def __init__(self):
+        self._smtp_username = os.environ["SMTP_USERNAME"]
+        self._smtp_password = os.environ["SMTP_PASSWORD"]
+        self._sheety_users_endpoint = os.environ["SHEETY_USERS_EP"]
```
Comment (lawjonathan17) [inline Folder/email_notifier.py:12]: nitpick butr you might wanna rename `SHEETY_USERS_EP` to `SHEETY_USERS_ENDPOINT`. Unless you can later remember EP stands for endpoint, which isn't a common abbreviation

File: Folder/email_notifier.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        users_response.raise_for_status()
+        self.users_data = users_response.json()["users"]
+        return self.users_data
+
+    def send_emails(self, user_email, message_body):
+
+        with smtplib.SMTP("smtp.gmail.com", port=587) as connection:
+            connection.starttls()
+            connection.login(user=self._smtp_username, password=self._smtp_password)
+            connection.sendmail(
```
Comment (lawjonathan17) [inline Folder/email_notifier.py:28]: you might wanna check if it sends the email correctly. Does this function return an error? You can add a print / log statement at the end, saying it was succesful

Comment (tokevinm) [reply Folder/email_notifier.py:28]: True, worked when I was testing but good idea to have one just to return a confirmation when successful

File: Folder/email_notifier.py (hunk 1 above)
Comment (michaelshum321) [reply Folder/email_notifier.py:7]: yea generally code docs in python are are done with `"""` 

```suggestion
class EmailNotifier:
    """
    All things related to Email
    """
    
    def __init__ ......
```

File: Folder/email_notifier.py (hunk 3 above)
Comment (michaelshum321) [reply Folder/email_notifier.py:28]: yea if I wrote this, after `sendmail` i would do
```
print(f"Sent email to {user_email}")
```

File: Folder/email_notifier.py (hunk 1 above)
Comment (tokevinm) [reply Folder/email_notifier.py:7]: Should've let you guys know but I made updates and merged it already with my main (added docstrings, etc), but didn't make it into a pull request. Well it was one initially but I merged it like pretty immediately just cause I made most of the recommended corrections, just not the ones where I replied to Jon with a question n he hadn't responded (at the time)

File: Folder/email_notifier.py (hunk 3 above)
Comment (tokevinm) [reply Folder/email_notifier.py:28]: I added an exception but it's pretty extra compared to that heh, probably didn't need to but wanted to use what I've been learning

**Pull request closed without merging**